### PR TITLE
Updating UVEX TLE

### PR DESCRIPTION
**Pull request by bwgref**
Have not adjusted the EarthLimbConstraint in the spirit of only changing one thing at a time.

### CODE REVIEW

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please fix the lint errors.

**Comment by bwgref** [comment]
Waiting on workflow approval to rerun checks

File: dorado/scheduling/mission.py
Comment (lpsinger) [inline dorado/scheduling/mission.py:168]: Which data directory? Not the data directory in this repository.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: dorado/scheduling/mission.py
Comment (bwgref) [reply dorado/scheduling/mission.py:168]: It was in the first commit. dorado/schedluing/data/uvex_tle_300_290_i30_72h.tle

Comment (lpsinger) [reply dorado/scheduling/mission.py:168]: I see. Well, all the TLEs are in the data directory, so this statement is a bit tautological.

Comment (lpsinger) [reply dorado/scheduling/mission.py:168]: Two requests:

1. Reword this to say simply, "The orbit is a snapshot TLE generated by an orbital simulation from SSL" or something similar. No need to mention the data directory.
2. If we are ready to fully replace the toy VELA orbit, then just delete the TLE and delete the bullet point that mentions it.

**Review by lpsinger — CHANGES_REQUESTED** [review]
Still one last lint error...

**Comment by mcoughlin** [comment]
Unit tests pass!

**Comment by lpsinger** [comment]
I'm really sorry, but I have to revert this because we haven't gotten @bwgref's CLA yet. Ughhh... lawyers.

**Comment by mcoughlin** [comment]
So we should be using this TLE locally for now, otherwise the science teams are building search efficiencies off of unrealistic schedules no?

**Comment by lpsinger** [comment]
I think we just need someone who has a signed CLA to provide the TLE.

**Pull request merged**
